src/scicookie/{{cookiecutter.project_slug}}/docs/contributing.md

```diff
@@ -110,127 +110,160 @@ If you are proposing a feature:
 - Remember that this is a volunteer-driven project, and that contributions are
   welcome :)
 
-## Get Started!
+## Get Started
 
 Ready to contribute? Here’s how to set up `{{ cookiecutter.project_slug }}` for local
 development.
 
-1.  Fork the `{{ cookiecutter.project_slug }}` repo on GitHub.
-1.  Clone your fork locally and change to the directory of your project:
+1. Fork the `{{ cookiecutter.project_slug }}` repo on GitHub.
+2. Clone your fork locally and change to the directory of your project:
+
 ```bash
-$ git clone git@github.com:your_name_here/{{ cookiecutter.project_slug }}.git
-$ cd {{cookiecutter.project_slug }}/
+git clone git@github.com:your_name_here/{{ cookiecutter.project_slug }}.git
```

Comment:
any reason for removing `$` ?

we usually add that to identify when the prompt line starts

Reply:
[MD014 - Dollar signs used before commands without showing output](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md014.md)